src/hammer-vlsi/defaults.yml

```diff
@@ -597,6 +619,20 @@ sram_generator.submit:
   settings: "vlsi.submit.settings"
   settings_meta: lazycrossref
 
+# PCB deliverable settings
```

Comment:
```suggestion
# PCB deliverable settings.
```

Reply:
I committed this just because I want to get this PR merged, but it doesn't make sense to put a period after this- it's not a sentence.